gratipay/models/account_elsewhere.py

```diff
@@ -271,7 +276,8 @@ def get_account_elsewhere(website, state, api_lookup=True):
             user_info = platform.get_user_info(key, uid)
         except Response as r:
             if r.code == 404:
-                _ = state['_']
+                # Prevent Content Spoofing by truncating long usernames
+                uid = (uid[:15] + '...') if len(uid) > 15 else uid
```

Comment:
Should be `:12` here if we're adding three characters. Could also try unicode ellipsis, which is only one character. ;-)